ndcube/ndcube.py

```diff
@@ -217,10 +217,10 @@ def crop(self,
 
         Examples
         --------
-        An example of cropping a region of interest on the Sun from a 3-D image-time cube:
-        >>> point1 = [SkyCoord(-50*u.deg, -40*u.deg, frame=frames.HeliographicStonyhurst), None]  # doctest: +SKIP
-        >>> point2 = [SkyCoord(0*u.deg, -6*u.deg, frame=frames.HeliographicStonyhurst), None]  # doctest: +SKIP
-        >>> NDCube.crop(point1, point2) # doctest: +SKIP
+        An example of cropping a region of interest on the Sun from a 3-D image-time cube
+            >>> point1 = [SkyCoord(-50*u.deg, -40*u.deg, frame=frames.HeliographicStonyhurst), None]  # doctest: +SKIP
+            >>> point2 = [SkyCoord(0*u.deg, -6*u.deg, frame=frames.HeliographicStonyhurst), None]  # doctest: +SKIP
+            >>> NDCube.crop(point1, point2) # doctest: +SKIP
```

Comment:
```suggestion
        An example of cropping a region of interest on the Sun from a 3-D image-time cube

        >>> point1 = [SkyCoord(-50*u.deg, -40*u.deg, frame=frames.HeliographicStonyhurst), None]  # doctest: +SKIP
        >>> point2 = [SkyCoord(0*u.deg, -6*u.deg, frame=frames.HeliographicStonyhurst), None]  # doctest: +SKIP
        >>> NDCube.crop(point1, point2) # doctest: +SKIP
```

Can you see if this works?

Reply:
yep tried it, it doesn't work